apollo-android-support/build.gradle.kts

```diff
@@ -22,7 +22,6 @@ android {
 dependencies {
   api(project(":apollo-normalized-cache-sqlite")) // TODO remove with removal of deprecated ApolloSqlHelper
   implementation(kotlin("stdlib"))
-  compileOnly(project(":apollo-runtime"))
-  compileOnly(project(":apollo-api"))
+  implementation(project(":apollo-runtime"))
```

Comment:
Not sure why this was `compileOnly` before

Reply:
This module has bunch of different stuff before. It included the Sqlite cache. Only reason that I can think of would be that some features work without runtime. But tbh, I don't think so. It is probably unnecessary.

Reply:
I think it was a mistake from day 0, might was thinking that without runtime you won't be able to use this module so dependency to runtime already was present in user's build configuration, but yeah no reason.